input/templates/partials/myaccount/my-address-book-adress.hbs

```diff
@@ -0,0 +1,18 @@
+<div>{{address.title}} {{address.firstName}} {{address.lastName}}</div>
```

Comment:
As this `address` represents the Address entity in our platform (http://dev.commercetools.com/http-api-types.html#address) we need to use the same names as it has. There is no `name` field, but we can build it by the combination of `title`, `firstName` and `lastName`.